scripts/genie.lua

```diff
@@ -1000,6 +1000,11 @@ end
 					"-Wno-extern-c-compat",
 				}
 			end
+      if (version >= 70000) then
```

Comment:
Perhaps add a comment here about apple using a different version with xcode 7 to prevent confusion in the future.